tests/cases/fourslash/completionsImport_ambient.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference path="fourslash.ts" />
+
+// @module: commonjs
+
+// @Filename: a.d.ts
+//// declare namespace foo { class Bar {} }
+//// declare module 'path1' {
+////   import Bar = foo.Bar;
+////   export default Bar;
+//// }
+//// declare module 'path2longer' {
+////   import Bar = foo.Bar;
+////   export {Bar};
+//// }
+////
+
+// @Filename: b.ts
+//// Ba/**/
+
+verify.applyCodeActionFromCompletion("", {
+  name: "Bar",
+  source: "path2longer",
```

Comment:
I thought the desired behaviour in the bug was to provide both path1 and path2longer. Does that test show this?

Reply:
Both options were provided, it's just that both of them resulted in the same code action, despite presenting differently. This test shows that the one that resolved to the wrong code action now resolves to the right code action, but doesn’t actually demonstrate that both completions are offered—I’ll add that so it’s more clear.